### ISSUE TITLE
Backtrace Error

### ISSUE THREAD

**Issue by crikapa7** [facebook/infer#943]
Error backtrace:
Raised at file "base/ProcessPool.ml", line 32, characters 6-72
Called from file "base/ProcessPool.ml" (inlined), line 35, characters 61-73
Called from file "backend/Tasks.ml", line 65, characters 4-36
Called from file "backend/InferAnalyze.ml", line 176, characters 8-36
Called from file "integration/Driver.ml", line 327, characters 2-74
Called from file "integration/Driver.ml" (inlined), line 412, characters 30-60
Called from file "backend/infer.ml", line 24, characters 2-47
Called from file "backend/infer.ml", line 112, characters 6-52
Uncaught error: ("InferModules__ProcessPool.Execution_error(\"died after receiving sigsegv (signal number 11)\")")

What does this error mean?

**Comment by jvillard** [comment]
This is a trace of execution from infer's main process, who is running multiple infer sub-processes in parallel. Looks like one of the infer sub-processes crashed with a segmentation fault.

To figure out what happened, could you provide more information?
- which version of infer are you using? (`infer --version`)
- which command did you run?
- please paste the full console output
- the contents of infer-out/logs could help too

**Comment by crikapa7** [comment]
- Infer version v0.13.1

- infer run -g -a checkers -- clang file.c

-


Starting analysis...

legend:
  "F" analyzing a file
  "." analyzing a procedure
  "C" analyzer crashed
  "T" timeout: procedure analysis took too much time
  "S" timeout: procedure analysis took too many symbolic execution steps
  "R" timeout: procedure analysis took too many recursive iterations

F
Error backtrace:
Raised at file "base/ProcessPool.ml", line 32, characters 6-72
Called from file "base/ProcessPool.ml" (inlined), line 35, characters 61-73
Called from file "backend/Tasks.ml", line 65, characters 4-36
Called from file "backend/InferAnalyze.ml", line 176, characters 8-36
Called from file "integration/Driver.ml", line 327, characters 2-74
Called from file "integration/Driver.ml" (inlined), line 412, characters 30-60
Called from file "backend/infer.ml", line 24, characters 2-47
Called from file "backend/infer.ml", line 112, characters 6-52
Uncaught error: ("InferModules__ProcessPool.Execution_error(\"died after receiving sigsegv (signal number 11)\")")

Run the command again with `--keep-going` to try and ignore this error.

I tried to use the --keep-going command but the analysis was not complete.

- Log file is too big to attach it here

**Comment by jvillard** [comment]
We managed to reproduce a similar crash and are investigating.

**Comment by emiraslan** [comment]
Anything new?

**Comment by jvillard** [comment]
Thanks for pinging. This part of infer has changed significantly since the time this issue was opened so I don't think this issue is still current. Please reopen (or file another issue) if you encounter this error again.
